app/views/lessons/index.html.erb

```diff
@@ -11,10 +11,12 @@
 
 <div class="row mt-5 mx-auto container">
         <% @category.each do |category| %>
-          <div class="card text-center ml-5 my-5" style="width: 18rem;">
+          <div class="card text-center mx-auto my-5" style="width: 18rem;">
                 <div class="card-header"><%= link_to category.title %></div>
                 <h5 class="card-title"><%= category.description %></h5>
-                <%= link_to "Start lesson", class: "btn btn-primary" %>
+                <div class="text-right">
+                <%= link_to "Start lesson", href="#", class: "btn btn-primary w-50" %>
+                </div>
           </div>
```

Comment:
to fix the design issue with the cards, you can use this code:
```
<div class="row">
      <% @category.each do |category| %>
      <div class="card text-center mx-auto my-3" style="width: 18rem;">
        <h5 class="card-header"><%= category.title %></h5>
        <div class="card-body">
          <h5 class="card-text"><%= category.description %></h5>
          <div class="text-right mt-3">
            <%= link_to "Start lesson", href="#", class: "btn btn-primary" %>
          </div>
        </div>
      </div>
      <% end %>
  </div>
  <div class="row d-flex justify-content-center">
    <%= will_paginate @category %>
  </div>
```
to make the number of items in each page more symmetrical/less crowded you can update/replace the `lessons_controller.rb` code to contain following:
` @category = Category.paginate(page: params[:page], per_page: 6 )`

end result will look something like this:
![image](https://user-images.githubusercontent.com/73863408/129501571-dfe2811a-8f72-4361-865c-b202b752a9c5.png)